### ISSUE TITLE
add captureWarning to dictConfig

### ISSUE THREAD

**Issue by Niccolum** [python/cpython#96829]
Allow user to configure [this](https://github.com/python/cpython/blob/main/Lib/logging/__init__.py#L2314) throught [DictConfigurator](https://github.com/python/cpython/blob/main/Lib/logging/config.py#L492)

**Comment by vsajip** [comment]
You'll need to give more detail about how you envisage this working. Note that you can use arbitrary callables in `dictConfig` to configure things.

**Comment by Niccolum** [comment]
For example, root key, like `version` or `disable_existing_loggers`

**Comment by vsajip** [comment]
That's not really enough detail, sorry. You haven't explained why you can't do this in a custom callable, and why it's even needed (given that there is an existing documented way to integrate `warnings` and `logging`).

**Comment by Niccolum** [comment]
Oh, sorry for the misunderstanding.

I use configuration like dict or file for logging in any my projects for work and open source.
Not so long ago I found out about [captureWarning](https://docs.python.org/3/library/logging.html#logging.captureWarnings).
In any my projects i tried to use structured format like json, but default warnings, like default logging, it's just string.
For better parsing warnings it's great, when it's look like logs. So, I wrote line more for captureWarning after dictConfig. So, I thought, it will be great, if this setting can be part of global config format
